src/pages/articles/canvas/basic-usage/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Basic Usage
+---
+## Basic Usage of Canvas
+
+When using canvas, first place a canvas into the document as an element.
+
+```
+<canvas width="400" height="400" id="canvas"></canvas>
+```
+
+The `width` and `height` attributes will control the size of the canvas. These attributes control the size of the drawing canvas, not the actual rendered size. See the "Canvas Dimensions" page for more.
+
+In order to use a `canvas`, we must first grab the element from the page as a DOM element, and then get a drawing context from it.
+
+```
+var canvas = document.getElementById("canvas");
+var ctx = canvas.getContext('2d');
+```
+
+All drawing calls after this will be made from the `ctx` object.
+
+There are a few available drawing contexts, including `webgl`. WebGL is really an entirely different technology, so we will only focus on 2D drawing.
```

Comment:
Can you link to WebGL? If the articles do not exist, open an issue to add them or feel free to open another PR to add them in. Running `node _normaliseArticles.js` will populate any new articles with a stub